[2024-12-03 14:44:40] local.ERROR: During class fetch: Uncaught ErrorException: include(C:\wael\DeliveryRequists\vendor\composer/../laravel/framework/src/Illuminate/Auth/GuardHelpers.php): Failed to open stream: No such file or directory in C:\wael\DeliveryRequists\vendor\composer\ClassLoader.php:576
Stack trace:
#0 C:\wael\DeliveryRequists\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(270): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'include(C:\\wael...', 'C:\\wael\\Deliver...', 576)
#1 C:\wael\DeliveryRequists\vendor\composer\ClassLoader.php(576): Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(2, 'include(C:\\wael...', 'C:\\wael\\Deliver...', 576)
#2 C:\wael\DeliveryRequists\vendor\composer\ClassLoader.php(576): include('C:\\wael\\Deliver...')
#3 C:\wael\DeliveryRequists\vendor\composer\ClassLoader.php(427): Composer\Autoload\{closure}('C:\\wael\\Deliver...')
#4 C:\wael\DeliveryRequists\vendor\tymon\jwt-auth\src\JWTGuard.php(24): Composer\Autoload\ClassLoader->loadClass('Illuminate\\Auth...')
#5 C:\wael\DeliveryRequists\vendor\composer\ClassLoader.php(576): include('C:\\wael\\Deliver...')
#6 C:\wael\DeliveryRequists\vendor\composer\ClassLoader.php(427): Composer\Autoload\{closure}('C:\\wael\\Deliver...')
#7 C:\wael\DeliveryRequists\vendor\tymon\jwt-auth\src\Providers\AbstractServiceProvider.php(95): Composer\Autoload\ClassLoader->loadClass('Tymon\\JWTAuth\\J...')
#8 C:\wael\DeliveryRequists\vendor\laravel\framework\src\Illuminate\Auth\AuthManager.php(113): Tymon\JWTAuth\Providers\AbstractServiceProvider->Tymon\JWTAuth\Providers\{closure}(Object(Illuminate\Foundation\Application), 'api', Array)
#9 C:\wael\DeliveryRequists\vendor\laravel\framework\src\Illuminate\Auth\AuthManager.php(90): Illuminate\Auth\AuthManager->callCustomCreator('api', Array)
#10 C:\wael\DeliveryRequists\vendor\laravel\framework\src\Illuminate\Auth\AuthManager.php(70): Illuminate\Auth\AuthManager->resolve('api')
#11 C:\wael\DeliveryRequists\vendor\laravel\framework\src\Illuminate\Auth\AuthManager.php(340): Illuminate\Auth\AuthManager->guard()
#12 C:\wael\DeliveryRequists\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(338): Illuminate\Auth\AuthManager->__call('id', Array)
#13 C:\wael\DeliveryRequists\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(341): Illuminate\Support\Facades\Facade::__callStatic('id', Array)
#14 C:\wael\DeliveryRequists\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(312): Illuminate\Foundation\Exceptions\Handler->context()
#15 C:\wael\DeliveryRequists\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(271): Illuminate\Foundation\Exceptions\Handler->buildExceptionContext(Object(ErrorException))
#16 C:\wael\DeliveryRequists\vendor\nunomaduro\collision\src\Adapters\Laravel\ExceptionHandler.php(46): Illuminate\Foundation\Exceptions\Handler->report(Object(ErrorException))
#17 C:\wael\DeliveryRequists\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(454): NunoMaduro\Collision\Adapters\Laravel\ExceptionHandler->report(Object(ErrorException))
#18 C:\wael\DeliveryRequists\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(157): Illuminate\Foundation\Console\Kernel->reportException(Object(ErrorException))
#19 C:\wael\DeliveryRequists\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 {main} {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): During class fetch: Uncaught ErrorException: include(C:\\wael\\DeliveryRequists\\vendor\\composer/../laravel/framework/src/Illuminate/Auth/GuardHelpers.php): Failed to open stream: No such file or directory in C:\\wael\\DeliveryRequists\\vendor\\composer\\ClassLoader.php:576
Stack trace:
#0 C:\\wael\\DeliveryRequists\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Bootstrap\\HandleExceptions.php(270): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleError(2, 'include(C:\\\\wael...', 'C:\\\\wael\\\\Deliver...', 576)
#1 C:\\wael\\DeliveryRequists\\vendor\\composer\\ClassLoader.php(576): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->Illuminate\\Foundation\\Bootstrap\\{closure}(2, 'include(C:\\\\wael...', 'C:\\\\wael\\\\Deliver...', 576)
#2 C:\\wael\\DeliveryRequists\\vendor\\composer\\ClassLoader.php(576): include('C:\\\\wael\\\\Deliver...')
#3 C:\\wael\\DeliveryRequists\\vendor\\composer\\ClassLoader.php(427): Composer\\Autoload\\{closure}('C:\\\\wael\\\\Deliver...')
#4 C:\\wael\\DeliveryRequists\\vendor\\tymon\\jwt-auth\\src\\JWTGuard.php(24): Composer\\Autoload\\ClassLoader->loadClass('Illuminate\\\\Auth...')
#5 C:\\wael\\DeliveryRequists\\vendor\\composer\\ClassLoader.php(576): include('C:\\\\wael\\\\Deliver...')
#6 C:\\wael\\DeliveryRequists\\vendor\\composer\\ClassLoader.php(427): Composer\\Autoload\\{closure}('C:\\\\wael\\\\Deliver...')
#7 C:\\wael\\DeliveryRequists\\vendor\\tymon\\jwt-auth\\src\\Providers\\AbstractServiceProvider.php(95): Composer\\Autoload\\ClassLoader->loadClass('Tymon\\\\JWTAuth\\\\J...')
#8 C:\\wael\\DeliveryRequists\\vendor\\laravel\\framework\\src\\Illuminate\\Auth\\AuthManager.php(113): Tymon\\JWTAuth\\Providers\\AbstractServiceProvider->Tymon\\JWTAuth\\Providers\\{closure}(Object(Illuminate\\Foundation\\Application), 'api', Array)
#9 C:\\wael\\DeliveryRequists\\vendor\\laravel\\framework\\src\\Illuminate\\Auth\\AuthManager.php(90): Illuminate\\Auth\\AuthManager->callCustomCreator('api', Array)
#10 C:\\wael\\DeliveryRequists\\vendor\\laravel\\framework\\src\\Illuminate\\Auth\\AuthManager.php(70): Illuminate\\Auth\\AuthManager->resolve('api')
#11 C:\\wael\\DeliveryRequists\\vendor\\laravel\\framework\\src\\Illuminate\\Auth\\AuthManager.php(340): Illuminate\\Auth\\AuthManager->guard()
#12 C:\\wael\\DeliveryRequists\\vendor\\laravel\\framework\\src\\Illuminate\\Support\\Facades\\Facade.php(338): Illuminate\\Auth\\AuthManager->__call('id', Array)
#13 C:\\wael\\DeliveryRequists\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Exceptions\\Handler.php(341): Illuminate\\Support\\Facades\\Facade::__callStatic('id', Array)
#14 C:\\wael\\DeliveryRequists\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Exceptions\\Handler.php(312): Illuminate\\Foundation\\Exceptions\\Handler->context()
#15 C:\\wael\\DeliveryRequists\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Exceptions\\Handler.php(271): Illuminate\\Foundation\\Exceptions\\Handler->buildExceptionContext(Object(ErrorException))
#16 C:\\wael\\DeliveryRequists\\vendor\\nunomaduro\\collision\\src\\Adapters\\Laravel\\ExceptionHandler.php(46): Illuminate\\Foundation\\Exceptions\\Handler->report(Object(ErrorException))
#17 C:\\wael\\DeliveryRequists\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Console\\Kernel.php(454): NunoMaduro\\Collision\\Adapters\\Laravel\\ExceptionHandler->report(Object(ErrorException))
#18 C:\\wael\\DeliveryRequists\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Console\\Kernel.php(157): Illuminate\\Foundation\\Console\\Kernel->reportException(Object(ErrorException))
#19 C:\\wael\\DeliveryRequists\\artisan(35): Illuminate\\Foundation\\Console\\Kernel->handle(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))
#20 {main} at C:\\wael\\DeliveryRequists\\vendor\\tymon\\jwt-auth\\src\\JWTGuard.php:24)
[stacktrace]
#0 {main}
"} 
